.github/workflows/snap-builds.yaml

```diff
@@ -0,0 +1,70 @@
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        required: true
+        type: string
+      toolchain:
+        required: true
+        type: string
```

Comment:
Unfortunately, [github doesn't support the choice type for reusable workflows](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_callinputsinput_idtype):


>## `on.workflow_call.inputs.<input_id>.type`
>
>Required if input is defined for the on.workflow_call keyword. The value of this parameter is a string specifying the data type of the input. This must be one of: boolean, number, or string.

But I definitely agree with your point that, since the reusable workflow expect certain values, those values should be made explicit as input. What would you think about listing the possible values in the description? That's definitely not as good as being able to use the `choice` type, but maybe an ok alternative?